plugins/mediaPlayer/mediaPlayer.ts

```diff
@@ -114,6 +127,8 @@ export const initMediaPlayer = (
 
         if (hasNext.value) {
           next();
+        } else if (queue.value.isRepeatEnabled) {
+          restart();
```

Comment:
Is it correct to show that there's no next track in the list? "HasNext" will be falsy when having reached the end of the list.

Reply:
good point. changed it